--- a/src/picker.rs	2025-09-01 11:25:57.725961319 +0100
+++ b/src/picker.rs	2025-09-01 11:27:39.264921598 +0100
@@ -44,11 +44,11 @@
 
 #[derive(Clone, Debug)]
 pub struct Picker {
-    font_size: FontSize,
-    protocol_type: ProtocolType,
-    background_color: Rgba<u8>,
-    is_tmux: bool,
-    capabilities: Vec<Capability>,
+    pub font_size: FontSize,
+    pub protocol_type: ProtocolType,
+    pub background_color: Rgba<u8>,
+    pub is_tmux: bool,
+    pub capabilities: Vec<Capability>,
 }
 
 /// Serde-friendly protocol-type enum for [Picker].
